metroae

```diff
@@ -86,6 +86,11 @@ fi
 if [[ -z $SETUP_FILE ]]; then
     SETUP_FILE=/opt/metroae/.metroae
 fi
+
+# figure out of this works!!!!!! (delete if block below if it doesn't!!!!!)
+if [[ -z $CHECK_CONTAINER_CONNECTION ]]; then
```

Comment:
what does this do?

Reply:
I was originally thinking of setting an environment variable in metroag.robot in the tests repo to use in the metroae script for testing (if this environment variable was set, then it would simulate having no internet from the container), but eventually went down a different route. I forgot to delete this.